examples/xed-examples-util.c

```diff
@@ -197,7 +197,7 @@ xed_int64_t xed_atoi_general(char* buf, int mul) {
     xed_int64_t b;
 
     char* p = buf;
-    while(*p && isspace(*p))
+    while(*p && isspace((unsigned char)*p))
```

Comment:
from NetBSD isspace(3) man page:
> CAVEATS
>      The argument to isspace() must be EOF or representable as an unsigned
>      char; otherwise, the behavior is undefined.  See the CAVEATS section of
>       ctype(3) for more details.

See also http://stackoverflow.com/questions/21805674/do-i-need-to-cast-to-unsigned-char-before-calling-toupper